Description: Fix FTBFS (kinda hackish) (Closes: #966888)
Author: Joachim Breitner <nomeata@debian.org>
Bug-Debian: https://bugs.debian.org/966888

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2020-08-28

--- unicode-screensaver-0.5.2.orig/hacks/screenhack.c
+++ unicode-screensaver-0.5.2/hacks/screenhack.c
@@ -130,7 +130,8 @@ extern struct xscreensaver_function_tabl
 
 
 const char *progname;   /* used by hacks in error messages */
-const char *progclass;  /* used by ../utils/resources.c */
+// const char *progclass;  /* used by ../utils/resources.c */
+const char *progclass = "Unicode";
 Bool mono_p;		/* used by hacks */
 
 
--- unicode-screensaver-0.5.2.orig/unicode.c
+++ unicode-screensaver-0.5.2/unicode.c
@@ -163,8 +163,6 @@ unicode_free (Display *dpy, Window windo
 }		
 
 
-char *progclass = "Unicode";
-
 static char const *unicode_defaults [] = {
   ".background: white",
   ".foreground: black",
